### PR TITLE
Feat: Added version info into sidebar

### LINKED ISSUE #343: Show version info in the velaux
Display at the bottom left of the page.

### PR DESCRIPTION
**Pull request by HendersonTyler**
### Description of your changes

Added version info from package.json to the sidebar

Fixes #343 

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary.
- [x] Run `yarn lint` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Comment by msftgits** [comment]
[CLA assistant check](https://cla.opensource.microsoft.com/oam-dev/velaux?pullRequest=344) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/oam-dev/velaux?pullRequest=344) before we can accept your contribution.<br/><br/>:x: HendersonTyler [sign now](https://cla.opensource.microsoft.com/oam-dev/velaux?pullRequest=344)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/oam-dev/velaux?pullRequest=344) it.</sub>

File: src/layout/LeftMenu/index.less [hunk 1]
```diff
@@ -90,3 +90,10 @@ a:hover {
     }
   }
 }
+
+.bottom {
+  position: absolute;
+  bottom: 10px;
+  width: 50%;
```
Comment (barnettZQG) [inline src/layout/LeftMenu/index.less:97]: 100% maybe is better?

**Comment by barnettZQG** [comment]
@HendersonTyler  Thank your first PR.

**Comment by HendersonTyler** [comment]
CSS has never been my favourite. 🥲 

Changing width to 50% didn't make the text align very well, so I changed a few more settings. Please let me know if it still isn't right.

File: src/layout/LeftMenu/index.less (hunk 1 above)
Comment (HendersonTyler) [reply src/layout/LeftMenu/index.less]: CSS has never been my favourite. 🥲

Changing width to 50% didn't make the text align very well, so I changed a few more settings. Please let me know if it still isn't right.

**Review by barnettZQG — APPROVED** [review]
LGTM

**Pull request merged**